### PR TITLE
Update eslint config

### PR DESCRIPTION
**Pull request by vpicone**
- Removes all the wesbos stuff
- Still uses Carbon prettier config (via eslint) for formatting

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
     "fathom": "readonly"
   },
+  "env": {
+    "node": true,
+    "browser": true,
+    "commonjs": true,
+    "es6": true
+  },
+  "parser": "babel-eslint",
+  "extends": [
```
Comment (joshblack) [inline .eslintrc:13]: Would it be helpful to try out: https://www.npmjs.com/package/eslint-config-carbon ? Might help with some of the base rules and then can layer on the import ones 👀

Comment (vpicone) [reply .eslintrc:13]: @joshblack for sure, can we update the peer dep to allow eslint 7?

Comment (joshblack) [reply .eslintrc:13]: Hell yeah 🔥

**Review by dakahn — APPROVED** [review]
Seems cool to me -- if we're still doing this and pending whatever other convos are going on

**Pull request closed without merging**